lametro/events.py

```diff
@@ -501,83 +505,66 @@ def find_approved_minutes(self, event):
         result = self.search(
             "/matters/",
             "MatterId",
-            "MatterBodyId eq {} and substringof('{}', MatterTitle) and substringof('Minutes', MatterTitle)".format(
-                event["EventBodyId"], date
+            (
+                f"MatterBodyId eq {event['EventBodyId']} and " +
+                f"substringof('{date}', MatterTitle) and " +
+                "(" +
+                    "(MatterTypeName eq 'Minutes') or " +
+                    "(substringof('Minutes', MatterTitle) and MatterTypeName eq 'Informational Report')" +
+                ")"
             ),
         )
 
-        try:
-            (matter,) = result
-        except ValueError as e:
-            if "not enough values" in str(e):
-                self.warning(
-                    "Couldn't find minutes for the {} meeting of {}.".format(name, date)
-                )
-                return None
-            elif "too many values to unpack" in str(e):
-                self.warning(
-                    "Found more than one minutes file for the {} meeting of {}.".format(
-                        name, date
-                    )
-                )
-                return None
-            else:
-                raise
-
-        attachment_url = self.BASE_URL + "/matters/{}/attachments".format(
-            matter["MatterId"]
-        )
+        # Will print a warning if no minutes have been found
+        valid_attachments = []
 
-        attachments = self.get(attachment_url).json()
+        # Sometimes, the search returns more than one board report.
+        # Go through each matter yielded from this generator to account for that.
+        for matter in result:
+            attachment_url = self.BASE_URL + "/matters/{}/attachments".format(
+                matter["MatterId"]
+            )
```

Comment:
>Note that the current implementation assumes that there will only ever be one board report with minutes in its attachments. Per this issue, there might be more than one, so we'll need to accommodate that.

Iterating through the generator - `result` - should account for multiple matters found